cb-tools/burgbox/+io/WSCommunicator.m

```diff
@@ -24,18 +24,14 @@
   
   properties (Transient)
     WebSocket
-    EventMode = false
+    EventMode = 'off'
```

Comment:
should these `false` -> `'off'` changes you've made be set with property validation (the `matlab.lang.OnOffSwitchState` ?)